### PR TITLE
fix: small updates in elastic indexer

### PR DESCRIPTION
**Pull request by azayz**
---

- [ ] This PR references an open issue
- [ ] Test added
- [ ] Documentation adjusted

### CODE REVIEW

File: now/executor/indexer/elastic/es_converter.py [hunk 1]
```diff
@@ -6,6 +6,40 @@
...
+            if (
+                k.startswith('embedding') or k.endswith('embedding')
+            ) and get_score_breakdown:
+                if 'embeddings' not in doc.tags:
+                    doc.tags['embeddings'] = {}
+                doc.tags['embeddings'][k] = v
+        da.append(doc)
+    return da
+
+
```
Comment (violenil) [inline now/executor/indexer/elastic/es_converter.py:42]: you may as well remove the ESConverter class and just have all the conversion helper functions in this file.

Comment (azayz) [reply now/executor/indexer/elastic/es_converter.py:42]: done

**Review by violenil — APPROVED** [review]
LGTM!

**Review by florian-hoenicke — APPROVED** [review]
nice

**Pull request merged**